api/v1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 IBM Corp.
```

Comment:
which year should we use?

Reply:
I used what was already in our `hack/boilerplate.go.txt` file

Reply:
When should we update it?

Reply:
I don't know if there is specific instructions about it

Reply:
I suggest opening a ticket for that.

Reply:
should it dynamically take the current year?

Reply:
What do you mean by dynamically? like an auto-generated code?

Reply:
could you write here the ticket number so I can resolve this?

Reply:
CSI-3239